money.go

```diff
@@ -0,0 +1,120 @@
+package money
+
+import "errors"
+
+var (
+	// ErrCurrencyMismatch happens when two compared Money don't have the same currency.
+	ErrCurrencyMismatch = errors.New("currencies don't match")
+)
+
+type Money struct {
+	Cents          int64   `json:"cents" bson:"cents"`
+	CurrencySymbol string  `json:"currency_symbol" bson:"currency_symbol"`
+	CurrencyIso    string  `json:"currency_iso" bson:"currency_iso"`
+	Label          string  `json:"label" bson:"label"`
```

Comment:
I think we need an getter for `label` if there isn't one, we might generate the label output with that